frontend/src/i18n/index.ts

```diff
@@ -0,0 +1,27 @@
+import i18n, { ResourceLanguage } from "i18next";
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+import enTranslations from "./locales/en.yaml";
+import deTranslations from "./locales/de.yaml";
+
+const resources = {
+    en: { translation: enTranslations as ResourceLanguage },
+    de: { translation: deTranslations as ResourceLanguage },
+};
+
+void i18n
+    .use(initReactI18next)
+    .use(LanguageDetector)
+    .init({
+        resources,
+        fallbackLng: "en",
+        interpolation: {
+            escapeValue: false,
+        },
+        detection: {
+            order: ["localStorage", "navigator"],
```

Comment:
Note that due to this order, there is currently no easy way to change one's language, other than manually clearing out `localStorage`. In particular, changing your browser language does not suffice.

Reply:
Yes, I know. There will be a language selector in the future and I think then this is certainly the order you want. As in: you might want to override the language for that specific website.